react/src/components/ResultsContainer.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import SearchInformation from './SearchInformation.jsx';
+import HitsList from './HitsList.jsx';
+
+const ResultsContainer = ({ result, navigateToPage }) => {
+
+    const isPrevButtonDisabled = () => {
+        if (result.page === 0)
```

Comment:
era mai simplu sa scrii:
`return result.page === 0;`

In cazul asta valoarea returnata coincide valoarea conditiei din `if`